pkg/kubelet/dockertools/fake_docker_client.go

```diff
@@ -206,11 +212,26 @@ func (f *FakeDockerClient) StartContainer(id string, hostConfig *docker.HostConf
 			Config:     &docker.Config{Image: "testimage"},
 			HostConfig: hostConfig,
 			State: docker.State{
-				Running: true,
-				Pid:     os.Getpid(),
+				Running:   true,
+				Pid:       os.Getpid(),
+				StartedAt: time.Now(),
 			},
 			NetworkSettings: &docker.NetworkSettings{IPAddress: "1.2.3.4"},
 		}
+		if f.ContainerMap != nil {
```

Comment:
I wish it were that simple. E.g. in `runonce_test` there's a testDocker class that 'inherits' (I don't know the proper word for go language) from FakeDockerClient.